packages/custom-functions-metadata/test/src/testCases.ts

```diff
@@ -48,7 +48,7 @@ describe("test cases", function() {
                         const expectedErrorsFile = path.join(testCaseDirPath, `expected.${scriptType}.errors.txt`);
                         const actualMetadataFile = path.join(testCaseDirPath, `actual.${scriptType}.json`);
                         const expectedMetadataFile = path.join(testCaseDirPath, "expected.json");
-                        const expectedMetadata: string | undefined = readFileIfExists(expectedMetadataFile);
+                        const expectedMetadata: string | undefined = readFileIfExists(expectedMetadataFile) || "";
```

Comment:
Why is the: ```|| ""``` needed here?

Reply:
If there are errors the actualMetadata will be an empty string now. Previously, it would have not written the file, so reading the file would return undefined.  Therefore, when there is an error, the actualMetadata will be an empty string, and if the expectedMetadataFile doesn't exist -- which it shouldn't -- then the expectedMetadata is also an empty string.